### PR TITLE
Hotfix lines breaks and for format date ISO 8601

### PR DESCRIPTION
**Pull request by killua99**
Issue with date ISO 8601 source: http://drupal.org/node/972532 patch http://drupal.org/node/972532#comment-5166548

And extra lines breaks and spaces.

### CODE REVIEW

File: includes/common.inc [hunk 1]
```diff
@@ -3221,6 +3224,10 @@ function drupal_common_theme() {
     'pager_link' => array(
       'arguments' => array('text' => NULL, 'page_new' => NULL, 'element' => NULL, 'parameters' => array(), 'attributes' => array()),
     ),
+    // from locale.inc
```
Comment (omega8cc) [inline includes/common.inc]: What is reasoning - issue # on d.o behind this?

**Comment by omega8cc** [comment]
Thanks for the patch. We may include it if you remove whitespace/newlines clean-up part, which should never be mixed with code changes, or it will break future merge attempts with upstream. Please re-roll the patch, separate only the related changes, with some comments / links to the issues on d.o. Thanks.

**Comment by killua99** [comment]
Now you can take a look for the commit, evaluate and commit this.

**Comment by omega8cc** [comment]
Looks good, thanks!

**Pull request merged**